src/components/signup-view/signup-view.jsx

```diff
@@ -16,8 +19,7 @@ export const SignupView = () => {
       Birthday: birthday,
     };
 
-    // Correctly place the fetch request inside the handleSubmit function
-    fetch("https://movie-api-lvgy.onrender.com", {
+    fetch("https://movie-api-lvgy.onrender.com/signup", {
```

Comment:
@allmyprogramming , this should be updated to `https://movie-api-lvgy.onrender.com/users` because /signup does not exist in your backend API so this fails...